src/bookshelf/Makefile.db

```diff
@@ -17,7 +17,7 @@ POSTGRES_CMD = sudo -u  postgres psql -U $(POSTGRES_USER)
 
 provision_test_db: $(PGPASSFILE)
 	$(POSTGRES_CMD) -c "CREATE DATABASE $(DB_NAME)"
-	$(POSTGRES_CMD) -c "CREATE USER $(DB_USER) SUPERUSER UNENCRYPTED PASSWORD '$(DB_PASS)';"
+	$(POSTGRES_CMD) -c "CREATE USER $(DB_USER) SUPERUSER PASSWORD '$(DB_PASS)';"
```

Comment:
Could you elaborate a bit more about this?

Reply:
See commit "UNENCRYPTED deprecated".

Reply:
```
    UNENCRYPTED deprecated

    from verify pipeline:

    make[1]: Leaving directory '/workdir/src/oc_bifrost'
    ===> Compiling bifrost
    ===> Dialyzer starting, this may take a while...
    ===> Updating plt...
    ===> Resolving files...
    ===> Updating base plt...
    ===> Resolving files...
    ===> Building with 196 files in "/root/.cache/rebar3/rebar3_22.2_plt"...
    ===> Copying "/root/.cache/rebar3/rebar3_22.2_plt" to "/workdir/src/oc_bifrost/_build/default/rebar3_22.2_plt"...
    ===> Checking 196 files in "/workdir/src/oc_bifrost/_build/default/rebar3_22.2_plt"...
    ===> Adding 106 files to "/workdir/src/oc_bifrost/_build/default/rebar3_22.2_plt"...
    ===> Doing success typing analysis...
    ===> Resolving files...
    ===> Analyzing 18 files with "/workdir/src/oc_bifrost/_build/default/rebar3_22.2_plt"...
    echo "localhost:5432:"bifrost_test":"bifrost_test_user":"bifrost_test_user_password"" > /workdir/src/oc_bifrost/.pgpass
    chmod 600 /workdir/src/oc_bifrost/.pgpass
    sudo -u  postgres psql -U 'postgres' -c "CREATE DATABASE "bifrost_test""
    CREATE DATABASE
    sudo -u  postgres psql -U 'postgres' -c "CREATE USER "bifrost_test_user" SUPERUSER UNENCRYPTED PASSWORD '"bifrost_test_user_password"';"
    ERROR:  UNENCRYPTED PASSWORD is no longer supported
    LINE 1: CREATE USER bifrost_test_user SUPERUSER UNENCRYPTED PASSWORD...
                                                    ^
    HINT:  Remove UNENCRYPTED to store the password in encrypted form instead.
    Makefile.db:19: recipe for target 'provision_test_db' failed
    make: *** [provision_test_db] Error 1
    <F0><9F><9A><A8> Error: The command exited with status 1

    ===============================================================

    make[1]: Leaving directory '/workdir/src/bookshelf'
    ===> Dialyzer starting, this may take a while...
    ===> Updating plt...
    ===> Resolving files...
    ===> Updating base plt...
    ===> Resolving files...
    ===> Building with 196 files in "/root/.cache/rebar3/rebar3_22.2_plt"...
    ===> Copying "/root/.cache/rebar3/rebar3_22.2_plt" to "/workdir/src/bookshelf/_build/default/rebar3_22.2_plt"...
    ===> Checking 196 files in "/workdir/src/bookshelf/_build/default/rebar3_22.2_plt"...
    ===> Adding 294 files to "/workdir/src/bookshelf/_build/default/rebar3_22.2_plt"...
    ===> Doing success typing analysis...
    ===> Resolving files...
    ===> Analyzing 20 files with "/workdir/src/bookshelf/_build/default/rebar3_22.2_plt"...
    ===> Verifying dependencies...
    ===> Compiling bookshelf
    make[1]: Entering directory '/workdir/src/bookshelf'
    grep: write error: Broken pipe
    grep: write error: Broken pipe
    make[1]: Leaving directory '/workdir/src/bookshelf'
    ===> Running cross reference analysis...
    echo "localhost:5432:bookshelf:bookshelf:test_sql_password" > /workdir/src/bookshelf/.pgpass
    chmod 600 /workdir/src/bookshelf/.pgpass
    sudo -u  postgres psql -U 'postgres' -c "CREATE DATABASE bookshelf"
    CREATE DATABASE
    sudo -u  postgres psql -U 'postgres' -c "CREATE USER bookshelf SUPERUSER UNENCRYPTED PASSWORD 'test_sql_password';"
    ERROR:  UNENCRYPTED PASSWORD is no longer supported
    LINE 1: CREATE USER bookshelf SUPERUSER UNENCRYPTED PASSWORD 'test_s...
                                            ^
    HINT:  Remove UNENCRYPTED to store the password in encrypted form instead.
    Makefile.db:19: recipe for target 'provision_test_db' failed
    make: *** [provision_test_db] Error 1
    <F0><9F><9A><A8> Error: The command exited with status 1
```